src/xml.ts

```diff
@@ -0,0 +1,52 @@
+import { XmlReplacements } from "./types";
+import { notDeepEqual } from "assert";
```

Comment:
It looks like this was accidentally imported at some point. I don't see it being used in here.

Reply:
Yeah not even sure how that happened. Fixed.